source/config.go

```diff
@@ -31,4 +31,7 @@ type Config struct {
 	// WaitTimeSeconds is the duration (in seconds) for which the call waits for
 	// a message to arrive in the queue before returning.
 	WaitTimeSeconds int32 `json:"aws.waitTimeSeconds" default:"10"`
+
+	// MaxNumberOfMessages is the maximum number of messages to return in a single batch.
```

Comment:
Nitpick: This description sounds to me like we're configuring the size of the batch returned by the connector to Conduit. I'd rather use the word "fetch" instead of "return", or even "fetch from SQS".